src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializerExt.cs

```diff
@@ -25,7 +25,7 @@ internal static partial class PrometheusSerializer
     {
         private static readonly string[] MetricTypes = new string[]
         {
-            "untyped", "counter", "gauge", "summary", "histogram", "histogram", "histogram", "histogram", "untyped",
+            "untyped", "counter", "gauge", "summary", "histogram", "histogram", "histogram", "histogram", "gauge",
```

Comment:
TODO: it might be good add a comment here to explain why. (like UpDownCounters are exported as Prometheus Gauge etc.)